### DISCUSSION TITLE
Support for Better Combat Mod

### DISCUSSION THREAD

**Discussion by ag120ah** [TechReborn/TechReborn#3033] [Ideas]
A small addition to add attack animations for the Nanosaber using the Better Combat Mod.

https://www.curseforge.com/minecraft/mc-mods/better-combat-by-daedelus

**Comment by Ayutac** [comment]
who is going to make the animation?

**Comment by xanthian** [comment]
No need its a sword which already exists in better combat, just need to add a json file for it
why it doesnt pick up the fact its an extended SwordItem i dunno...


data\techreborn\weapon_attributes\nanosaber.json
`
{
  "parent": "bettercombat:sword"
}
`

problem is because of insane attack speed on it, it looks terrible.

**Reply by Ayutac** [reply]
not like ninja-fast?

**Reply by xanthian** [reply]
yes ninja fast.
try it.
